gpii/node_modules/flowManager/test/BrowserChannelTests.js

```diff
@@ -0,0 +1,400 @@
+/**
+ * GPII Flow Manager Browser Channel Tests
+ *
+ * Copyright 2013 OCAD University
+ * Copyright 2015 Emergya
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/kettle/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion"),
+    path = require("path"),
+    jqUnit = fluid.require("jqUnit"),
+    configPath = path.resolve(__dirname, "./configs"),
+    kettle = require("kettle"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.require("ontologyHandler", require);
```

Comment:
All these 6 includes can just be replaced by 
require("universal") - this will ensure that this test fixture can be run individually, removing the "universal is not loaded" error

This should be done everywhere else too - fluid.require doesn't really have a function any more, unless you are trying to load a module-qualified path, it is identical to plain require